package.json

```diff
@@ -175,6 +177,8 @@
     "pino-v6": "npm:pino@^6.13.3",
     "pino-v7": "npm:pino@^7.8.1",
     "pkg-up": "^3.1.0",
+    "prebuildify": "^5.0.1",
```

Comment:
Suggestion: `prebuildify` version 6.0.0  released, can we use the latest? https://www.npmjs.com/package/prebuildify/v/6.0.0

Reply:
Ah, they released a new major version during my work. I don't prefer to jump to a X.0.0 release. And the changes do not look interesting to me.